src/gluonts/distribution/neg_binomial.py

```diff
@@ -130,7 +130,9 @@ def distribution(
         else:
             F = getF(mu)
             mu = F.broadcast_mul(mu, scale)
-            alpha = F.broadcast_add(alpha, F.broadcast_div(scale - 1, mu))
+            alpha = F.broadcast_add(
```

Comment:
OK, so isn't the scale is already multiplied into `mu` on line 132?

Reply:
Of course, that was me being tired